src/sentry/incidents/endpoints/serializers.py

```diff
@@ -306,6 +311,21 @@ def validate_aggregation(self, aggregation):
                 % [item.value for item in QueryAggregations]
             )
 
+    def validate_environment(self, environment):
+        def convert_to_environment(env_name):
+            try:
+                return Environment.objects.get(
+                    organization_id=self.context["organization"].id, name=env_name
+                )
+            except Environment.DoesNotExist:
+                raise serializers.ValidationError("Environment is not part of this organization")
```

Comment:
I guess you already addressed this as not being ideal since it pretty much exactly replicates EnvironmentField::to_internal_value. I guess given the temporary nature it didn't feel worth the effort to do something better? Like having the `EnvironmentField` be used as top-level instead of the list child (instead of "ObjectField") and modifying `to_internal_value` to do the conversion from single env to list if necessary.

Reply:
Yeah, I didn't really think it was worth doing this well, since we'll just have `EnvironmentField` top level soon anyway.